src/Azalee/FactionLegends/API/FactionLegendsAPI.php

```diff
@@ -34,104 +35,78 @@ class FactionLegendsAPI
 
     public function loadData(FactionLegends $plugin): void
     {
-        $plugin->getDatabase()->executeSelect(QuerysInterface::FactionLegends_Faction_Load, [], function (array $row): void
-        {
-            foreach($row as $rows)
-            {
-                $this->factions =
-                    [
-                        "name" => $rows["name"],
-                        "description" => $rows["description"],
-                        "status" => $rows["status"],
-                        "players" => $rows["players"],
-                        "power" => $rows["power"],
-                        "money" => $rows["money"],
-                        "allies" => $rows["allies"],
-                        "claims" => $rows["claims"]
-                    ];
-            }
-        });
-        $plugin->getDatabase()->executeSelect(QuerysInterface::FactionLegends_Player_Load, [], function (array $row): void
-        {
-            foreach($row as $rows)
-            {
-                $this->players =
-                    [
-                        "name" => $rows["name"],
-                        "faction" => $rows["faction"],
-                        "role" => $rows["role"]
-                    ];
-            }
-        });
-        $plugin->getDatabase()->executeSelect(QuerysInterface::FactionLegends_Home_Load, [], function (array $row): void
-        {
-            foreach($row as $rows)
-            {
-                $this->home =
-                    [
-                        "name" => $rows["name"],
-                        "faction" => $rows["faction"],
-                        "x" => $rows["x"],
-                        "y" => $rows["y"],
-                        "z" => $rows["z"],
-                        "world" => $rows["world"]
-                    ];
-            }
-        });
-        $plugin->getDatabase()->executeSelect(QuerysInterface::FactionLegends_Lang_Load, [], function (array $row): void
-        {
-            foreach($row as $rows)
-            {
-                $this->lang =
-                    [
-                        "name" => $rows["name"],
-                        "lang" => $rows["lang"]
-                    ];
+        $plugin->getDatabase()->executeSelect(QuerysInterface::FactionLegends_LoadData, [], function (array $rows): void {
```

Comment:
QuerysInterface::FactionLegends_LoadData is undefined

Reply:
yes QuerysInterface::FactionLegends_LoadData because it has to be done in the mysql.sql and sqlite.sql files